torch/_tensor.py

```diff
@@ -96,7 +96,14 @@ def __deepcopy__(self, memo):
             # doesn't work because of
             # https://github.com/pytorch/pytorch/issues/47442
             # Update the test in test_serialization if you remove 'meta' from here
-            if self.is_sparse or self.device.type in ['lazy', 'xla', 'mps', 'ort', 'meta', 'hpu'] or \
+
+            fake_tensor_dispatch = isinstance(
```

Comment:
Can we just override `__deepcopy__` on the FakeTensor instead of modifying `__deepcopy__` on torch.Tensor itself?

Reply:
They're not FakeTensors at this point, we're deepcopying the original regular Tensors to FakeTensors.